README.md

```diff
@@ -580,6 +576,95 @@ This document is written to capture the conventions of writing styles for our pr
 ### Disclaimer
 It's important to note that client and project requirements always undo choices made in these guidelines
 
+### Formatting and Naming
+
+Names should be meaningful and not presentational:
+```css
+/* Bad examples */
+
+/* presentational with color and position reference */
+.button-orange {}
+.left-block {}
+
+/* meaningless */
+.yee-u88 {}
+```
+Some good examples would be:
+```css
+.video {}
+
+.gallery {}
+
+```
+
+Selectors are placed on separate lines:
+```css
+/* Recommended */
+h1,
```

Comment:
use classnames instead of selectors (see below)?

Reply:
There has been an exception added that would make this example valid. Also I would prefer to keep the example here simple. A usage of different classNames might send a message that looks like extending styles.